python/doc/examples/functional_modeling/univariate_functions/plot_createUnivariateFunction.py

```diff
@@ -0,0 +1,136 @@
+#
+# - Case 1: using an orthogonal polynomials family,
+# - Case 2: using a univariate polynomials,
+# - Case 3: using an orthogonal functions family.
+#
+#
+# Case 1: we use an orthogonal polynomials family
+# ===============================================
+#
+# In that case, the polynomials are orthogonal with respect to a measure.
+#
+# For example: we consider the Legendre polynomials family, orthogonal with respect to the Uniform distribution. We use the :class:`~openturns.LegendreFactory` class.
+# The method :meth:`~openturns.LegendreFactory.build` returns the polynomial number :math:`k` of the family.
+
+f1 = ot.LegendreFactory().build(1)
+f2 = ot.LegendreFactory().build(2)
+f3 = ot.LegendreFactory().build(3)
+
+print(type(f1))
+
+g = f1.draw(-1.0, 1.0, 256)
+g.add(f2.draw(-1.0, 1.0, 256))
+g.add(f3.draw(-1.0, 1.0, 256))
+g.setLegends([r'$\phi_1(x)$', r'$\phi_21(x)$', r'$\phi_3(x)$'])
+g.setLegendPosition('bottomright')
+g.setColors(ot.Drawable.BuildDefaultPalette(3))
+g.setTitle('Legendre Polynomials')
+View(g).show()
+
+# %%
+# We get the measure associated to the polynomials family:
+measure = ot.LegendreFactory().getMeasure()
+
+# %%
+# Case 2: we use a univariate polynomial
+# ======================================
+#
+# In that case, the polynomials are not necessarily orthogonal with respect to a measure.
+#
+# We can use:
+#
+# - the :class:`~openturns.UniVariatePolynomial` class,
+# - the :class:`~openturns.MonomialFunctionFactory` class,
+# - the :class:`~openturns.MonomialFunction` class.
+#
+# For example, we consider :
+#
+# .. math::
+#
+#     \begin{array}{lcl}
+#       f(x) & = & 1+2x+3x^2+4x^3\\
+#       g(x) & = & x^3
+#     \end{array}
+
+f = ot.UniVariatePolynomial([1.0, 2.0, 3.0, 4.0])
+g1 = ot.MonomialFunctionFactory().build(3)
+g2 = ot.MonomialFunction(3)
+print(f)
+print(g1)
+print(g2)
+
+# There is no associated measure
+# print(ot.MonomialFunctionFactory().getMeasure())
+
+# %%
+# Case 3: we use an orthogonal functions family
+# =============================================
+#
+# In that case, the function are orthogonal with respect to a measure :math:`\mu`.
+#
+# We can use:
+#
+# - the :class:`~openturns.HaarWaveletFactory` class,
+# - the :class:`~openturns.FourierSeriesFactory` class.
+#
+# The method :meth:`~openturns.HaarWaveletFactory.build` returns the function number :math:`k` of the family.
+
+# For example, we consider a Haar Wawelet:
+f1 = ot.HaarWaveletFactory().build(1)
+f2 = ot.HaarWaveletFactory().build(2)
+f3 = ot.HaarWaveletFactory().build(3)
+
+g = f1.draw(0.0, 1.0, 256)
+g.add(f2.draw(0.0, 1.0, 256))
+g.add(f3.draw(0.0, 1.0, 256))
+g.setLegends([r'$\phi_1(x)$', r'$\phi_21(x)$', r'$\phi_3(x)$'])
+g.setLegendPosition('topright')
+g.setColors(ot.Drawable.BuildDefaultPalette(3))
+g.setTitle('Haar Wavelets')
+View(g).show()
+
+# We get the measure: for the Haar Wavelet family, the Uniform(0,1)
+measure = ot.HaarWaveletFactory().getMeasure()
+
+# For example, we consider a Fourier Series:
```

Comment:
The code block starting with this line which goes all the way to the end of the script should be split, and the comments should appear as regular text IMO.

Reply:
ok.